comfy_cli/config_manager.py

```diff
@@ -43,6 +43,27 @@ def get(self, key):
         """
         return self.config["DEFAULT"].get(key, None)  # Returns None if the key does not exist
 
+    def get_or_override(self, env_key: str, config_key: str, set_value: Optional[str]) -> str | None:
```

Comment:
I am going to self-request one change. Let's change this from `set_value: Optional[str]` to `set_value: Optional[str] = None`. That way, the function can be used more conveniently from code locations where an environment override is desired and there is no corresponding `--set-X` Comfy-CLI flag value.

Reply:
Resolved in the new commit. I'm leaving this review open for visibility.